src/pages/homeLayout/cookiePopupWindow/cookiePopupWindow.tsx

```diff
@@ -1,29 +1,38 @@
 "use client";
 
-import Image from "next/image";
-import Link from "next/link";
 import { useState } from "react";
+
 import cn from "classnames";
 
 import Button from "../../../ui/button/button";
-import { MainPath } from "../../../services/router/routes";
 import { cookieLinkTitle } from "../../../mocks/cookiePopupWindow";
 
-import closeIcon from "../../../assets/pages/mainPage/closeIcon.svg";
+import useAppSelector from "../../../hooks/useAppSelector";
+import cookieStatusSelector from "../../../services/redux/features/cookieStatus/cookieStatusSelector";
+import useAppDispatch from "../../../hooks/useAppDispatch";
+import { setCookieStatus } from "../../../services/redux/features/cookieStatus/cookieStatusSlice";
 
 import styles from "./cookiePopupWindow.module.scss";
 
 const CookiePopupWindow = () => {
```

Comment:
can you remane this page component from "CookiePopupWindow" to "CookiePopupWindowPage"?

Reply:
what do you mean here?

Reply:
const CookiePopupWindow = () => {
=>
const CookiePopupWindowPage= () => {
just rename page component

Reply:
it's already done. It seemed like the same conversation twice